code/core/src/main/java/com/adobe/marketing/mobile/EventSource.java

```diff
@@ -45,4 +45,5 @@ private EventSource() {}
             "com.adobe.eventSource.errorResponseContent";
     public static final String CREATE_TRACKER = "com.adobe.eventSource.createTracker";
     public static final String TRACK_MEDIA = "com.adobe.eventSource.trackMedia";
+    public static final String CONTENT_COMPLETE = "com.adobe.eventSource.contentComplete";
```

Comment:
Can you also run `make api-dump` for updating the core.api file.

Reply:
👍